packages/@vuepress/core/lib/app/components/LayoutDistributor.js

```diff
@@ -0,0 +1,19 @@
+import Vue from 'vue'
+import { getLayoutAsyncComponent } from '../util'
+
+Vue.component('Layout', getLayoutAsyncComponent('Layout'))
+Vue.component('NotFound', getLayoutAsyncComponent('NotFound'))
```

Comment:
I propose to move these two `Vue.component()` lines to `app.js`.

What do you think about that? @ulivz